pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -759,6 +771,10 @@ func Run() {
 		case change := <-subDevicePortConfigList.C:
 			subDevicePortConfigList.ProcessChange(change)
 
+		case change := <-subAppFlowMonitor.C:
+			log.Infof("FlowStats: change called")
```

Comment:
May be we can make this log.Debugf instead of log.Infof

Reply:
sure. will change to debug.